# TF Parameters
# translations in milimeters
# rotations in degrees
# necessary conversion is being made in tf_broadcaster.cpp

laser_tran_x: 0.0
laser_tran_y: 0.0
laser_tran_z: 70.0
laser_rot_x: 0.0
laser_rot_y: 0.0
laser_rot_z: 0.0

#camera_tran_x: 107.50
#camera_tran_y: 0.0
#camera_tran_z: 140.0
#camera_rot_x: 0.0
#camera_rot_y: 30.0
#camera_rot_z: 0.0

